lib/screens/explore_screens/swap_seeds/interactor/viewmodels/swap_seeds_state.dart

```diff
@@ -1,15 +1,15 @@
 part of 'swap_seeds_bloc.dart';
 
 class SwapSeedsState extends Equatable {
-  final PageCommand? pageCommand;
+  final NavigateToSendConfirmation? pageCommand;
```

Comment:
adding stronger typing everywhere so we don't have to keep testing everything. We always know it's that type of page command, and no other type (which is true at this point)